contracts/truefi2/TrueLender2.sol

```diff
@@ -382,8 +382,7 @@ contract TrueLender2 is ITrueLender2, UpgradeableClaimable {
         uint256 borrowSum;
         uint256 resultPrecision = uint256(10)**decimals;
 
-        // loop through loans and sum amount borrowed accounting for precision
-        ITrueFiPool2[] memory tvlPools = rateAdjuster.getTVLPools();
+        ITrueFiPool2[] memory tvlPools = factory.getSupportedPools();
```

Comment:
Apologies for my duplicate work in PR #934. I hadn't realized you were also planning to do these changes in this task.

Some lessons for both of us for the future:
- If there's a good chance someone else is doing the same work, I should message the task owner to double-check we're not duplicating, before I start.
- If a task starts expanding to include a lot of unrelated changes, it might be a good idea to break it into smaller tasks (easier to review, less conflict with other PRs, shorter task cycle, better team visibility).

Reply:
It looks like we can revert the `TrueLender2.sol` changes in this PR. Renaming from `pools` to `tvlPools` isn't necessary since we already know in context they're coming from `getSupportedPools()`.